pkg/restore/restore.go

```diff
@@ -456,6 +460,80 @@ func (ctx *context) execute() (Result, Result) {
 		}
 	}
 
+	// TODO: Re-order this logic so that CRs can be prioritized in the main loop, rather than after.
+
+	// Refresh and resolve based on CRDs added to the API server from the above restore loop.
+	// This is because CRDs have been added to the API groups but until we refresh, Velero doesn't know about the
+	// newly-added API groups in order to create the CRs from them.
+	if err := ctx.discoveryHelper.Refresh(); err != nil {
+		addVeleroError(&errs, errors.Wrap(err, "error refreshing discovery API"))
```

Comment:
since we're not breaking here if we get an error, we'll still fall through to the rest of the logic - I *think* that's fine, since `newResources` will end up being the same as `ctx.prioritizedResources` so nothing new will be restored - that sound right?

Reply:
Yeah, that should be correct. I'll add a comment explaining this.

I am going to update the error to be `errors.WithStack` instead of `errors.Wrap` since it's not a Velero error, too.

Reply:
AFAIK `errors.Wrap(...)` will add a stack trace too, so it should be fine as-is

Reply:
Whoops, you're right. I'll add that back.